src/main/containers/Panels.jsx

```diff
@@ -2,12 +2,17 @@ import React, { useState } from 'react';
 import SourcePanel from './SourcePanel.jsx';
 import TestPanel from './TestPanel.jsx';
 import DestinationPanel from './DestinationPanel.jsx';
+import styled from 'styled-components';
```

Comment:
let move this above all .jsx component imports. Its common practice to import your libraries first and then your component.

Reply:
Done ✅